src/System.CommandLine/Command.cs

```diff
@@ -151,34 +155,44 @@ public override IEnumerable<CompletionItem> GetCompletions(CompletionContext con
 
             if (context.WordToComplete is { } textToMatch)
             {
-                var commands = Subcommands;
-                for (int i = 0; i < commands.Count; i++)
+                if (HasSubcommands)
                 {
-                    AddCompletionsFor(commands[i]);
+                    var commands = Subcommands;
```

Comment:
For all the `var`s that don't comply with dotnet/runtime guidelines, are you planning on addressing them all at once in the future? 
```suggestion
                    IList<Command> commands = Subcommands;
```

Reply:
We need to discuss it with other contributors. And the usage of `is { }` instead `is not null` which drives me crazy ;p